apps/nextjs-website/src/app/layout.tsx

```diff
@@ -4,6 +4,7 @@ import 'swiper/css';
 import 'swiper/css/navigation';
 import 'swiper/css/pagination';
 import '@/styles/globals.css';
+import '@/polyfill';
```

Comment:
Why is there this import here?

Reply:
Because i need to import the polyfill in this file to solve this [problem](https://github.com/char0n/swagger-ui-nextjs/issues/1) .

FYI: I used this [solution](https://github.com/swagger-api/swagger-ui/issues/8245#issuecomment-1718801519)